lib/davinci_dtr_test_kit/client_groups/full_ehr/dtr_full_ehr_questionnaire_response_correctness_test.rb

```diff
@@ -1,34 +1,36 @@
 require_relative '../../dtr_questionnaire_response_validation'
+require_relative '../../cql_test'
 
 module DaVinciDTRTestKit
   class DTRFullEHRQuestionnaireResponseCorrectnessTest < Inferno::Test
     include DTRQuestionnaireResponseValidation
+    include DaVinciDTRTestKit::CQLTest
 
     id :dtr_full_ehr_questionnaire_response_correctness
     title 'QuestionnaireResponse is correct for the workflow'
     description %(
-      The QuestionnaireResponse aligns with the following expectations for the workflow. This includes checks for
-      the presence of the following answers and their appropriate origin source extensions:
-
-      - `PBD.1` (Last Name) and `LOC.1` (Location): `auto`
-      - PBD.2 (First Name): `override`
-      - `3` (all nested dinner questions): `manual`
+      This test verifies that all the QuestionnaireResponse.item have the appropriate
+      origin source extensions and that all required questions have been answered.
     )
 
     run do
-      skip_if questionnaire_response.nil?, 'Completed QuestionnaireResponse input was blank'
+      skip_if questionnaire_response.blank?, 'Completed QuestionnaireResponse input was blank'
       check_is_questionnaire_response(questionnaire_response)
 
       qr = FHIR.from_contents(questionnaire_response)
-      questionnaire = if config.options[:adaptive]
-                        qr.contained.find do |res|
-                          res.resourceType == 'Questionnaire'
-                        end
-                      else
+      questionnaire = if scratch[:static_questionnaire_bundles].nil?
```

Comment:
Is the inputted custom questionnaire bundle supposed to be in scratch here? It doesn't seem to be finding it for me

Reply:
yeah, they don't all run together, so scratch does not persist across groups in this case. Updated the logic, adding the custom response input to this test again